frontend/yarn.lock

```diff
@@ -16058,12 +16058,7 @@ minimist-options@4.1.0:
     is-plain-obj "^1.1.0"
     kind-of "^6.0.3"
 
-minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
-  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
```

Comment:
Why was this deleted?

Reply:
Because the matching pattern below has been updated to all resolve to 1.2.6 (e.g. minimist @ 1.1.1, 1.1.3, ^1.2.0, ^1.2.5 all resolve the the version 1.2.6

Reply:
Oh oops, I should've read more closely. Thanks!

Reply:
no worries, lockfiles are very odd :D